sync/versions.py

```diff
@@ -14,50 +14,94 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import click
 import copy
 import logging
+import os
+import sync
 import sys
 
-from absl import app
-from absl import flags
 
-import sync
+DEFAULT_CONFIG_FOLDER = os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), 'config')
+
+
+class VersionNotFoundError(Exception):
+    pass
+
+
+@click.group()
+def versions():
+    pass
+
+
+@versions.command()
+@click.option('--config-folder', default=DEFAULT_CONFIG_FOLDER,
+              help='the folder that contains the config files')
+@click.option('--project', default="pipeline",
```

Comment:
/lgtm
NIT: This looks good to me. I don't know how I feel about having pipelines as a default. I understand that it is basically the core of tekton. However, I found myself accidentally adding versions to tekton because I forgot the --project flag. It would be better if we threw an error rather than having a default value.

Reply:
That's a good point, I will change the logic there.